external-api/src/constants/eStore.constant.ts

```diff
@@ -1,6 +1,16 @@
+export const ENDPOINT = {
+  SITE: 'sites',
+  TERM: 'terms/facilities',
+  BUYER: 'buyer',
+  DEAL: 'deals',
+  FACILITY: 'facilities',
+  DOCUMENT: 'documents',
+};
+
```

Comment:
To make this more enum-y could we have 

```suggestion
export const ENDPOINT = {
  SITE: 'sites',
  TERM: 'terms/facilities',
  BUYER: 'buyer',
  DEAL: 'deals',
  FACILITY: 'facilities',
  DOCUMENT: 'documents',
} as const;
```

to make these fields read only please -- this should help with tightening the typescript type for this (as apposed to `ENDPOINT.SITE: string` it would be `ENDPOINT.SITE: 'sites'`)

We can then do
```
export type EndpointType = ValuesOf<typeof ENDPOINT>;
```

to get an auto updating type for it

Reply:
Have made `ENDPOINT` as a type with `as const` however I don't think `ValuesOf` script should be to enforce typechecking as out of the box `type` does the job. Looking into the code it appears `ValuesOf` is an external function included in `libs/common` in my personanl opinion I rather prefer using out of the box TS `type` keyword rather than an external site help function which seems to be a workaround.

As suggested `as const` is a solution I have added for immutability and readonly values.